src/SupervisionSuccess/components/Outcomes/Outcomes.js

```diff
@@ -38,20 +45,32 @@ const Outcomes = ({ isError, prisonPopulationDiff, savings }) => {
     <div className="outcomes">
       <div className="outcomes_heading">
         Outcomes
-        <p>From 2019 baseline</p>
+        <p>From {year} baseline</p>
       </div>
       <div className="outcomes_stats">
         <div className="outcomes_stat">
           <div className={`outcomes_stat-value ${prisonPopulationDiffIconClass}`}>
             {isError ? "-" : Math.abs(prisonPopulationDiff)}
           </div>
           <div className="outcomes_stat-key">{prisonPopulationDiffText}</div>
+          <div className="outcomes_stat-proportion">
+            Revocations: {revocationsProportion || "-"}%
```

Comment:
This is discussed in the ticket, but we don't actually want to fall back to "-" in the case where one of the proportions is responsible for all of the change. The rule is: if one of those numbers is going to be "100%" then don't even display the other line. For example, in the case below, we shouldn't even show the line for revocations:
<img width="350" alt="Screen Shot 2021-07-16 at 11 17 22 AM" src="https://user-images.githubusercontent.com/431895/125985732-f9deda4a-34e9-4ae5-8018-327bbb168676.png">

Reply:
done

Reply:
@nasaownsky it doesn't look like you've pushed up a new commit, so I can't verify the fix just yet. Is that coming soon?

Reply:
@jessex Forgot to push, sorry. Here it is.

Reply:
It looks like this was fixed in one place but not the other:
<img width="1280" alt="Screen Shot 2021-07-20 at 4 29 08 PM" src="https://user-images.githubusercontent.com/431895/126403609-1647c9c1-c5dc-43a9-a29b-f3493a949c5b.png">

Reply:
Yeah, this is because the question I asked below.